tests/lib/rules/render-result-naming-convention.test.ts

```diff
@@ -97,11 +97,24 @@ ruleTester.run(RULE_NAME, rule, {
           const button = screen.getByText('some button');
         });
       `,
-      options: [
-        {
-          renderFunctions: ['customRender'],
-        },
-      ],
+      settings: {
+        'testing-library/custom-renders': ['customRender'],
+        'testing-library/custom-modules': ['test-utils'],
+      },
+    },
+    {
+      code: `
```

Comment:
There is a case not covered in tests which I forgot to bring here. What about these 2 cases?


```javasacript
    {
      code: `
        import { render } from '@testing-library/foo';

       const setup = () => render(<SomeComponent />);
        
        test('should not report straight destructured render result from custom module', () => {
          const { unmount } = setup();
        });
      `,
      settings: {
        'testing-library/custom-renders': ['setup'],
      },
    },
```

```javasacript
    {
      code: `
        import { customRender } from 'test-utils';
        
        test('should not report straight destructured render result from custom module', () => {
          const { unmount } = customRender(<SomeComponent />);
        });
      `,
      settings: {
        'testing-library/custom-renders': ['customRender'],
      },
    },
```

On the first example: "custom-renders" option could be understood as user wants to report `setup` as `render` alias, while it's not imported from TL. So `getTestingLibraryRenderImport` would return just "render" which is the imported render found, but in the rule implementation we would need to still look for those custom renders. I don't think we can automate this somehow with another util to find custom renders not directly imported. I think the most we can do is to add a new util method where you pass the node containing "setup" and found render method returned by `getTestingLibraryRenderImport` and the method just checks if "setup" (or whatever custom render received) is wrapping given render. This sounds to much for me, I'd leave this up to each rule implementation.

On the second example: "custom-renders" is setup but there is no "custom-modules" setup. Our new util methods won't be aware of this "customRender" as there is no custom module where it could be possibly imported. I think this is fine.